styleguide/source/_patterns/01-molecules/06-components/11-hub-card/hub-card-portrait.twig

```diff
@@ -0,0 +1,8 @@
+{#
+ # @file hub-card-portrait.twig
+ # Copyright 2017 Palantir.net, Inc.
+ #}
+
+<a class="hub-card hub-card-portrait hub-card-image-123" href="#">
```

Comment:
Ok, so doing more testing with the rows included I don't see how this can't be a background image... otherwise there'll just be a ton of white space. Will review with Andy, but for now this looks good to me.